boards/shields/x_nucleo_idb05a1/Kconfig.shield

```diff
@@ -0,0 +1,6 @@
+# Copyright (c) 2019 Linaro Limited
+# SPDX-License-Identifier: Apache-2.0
+
+config SHIELD_X_NUCLEO_IDB05A
+	bool "ST BLE shield with BlueNRG"
```

Comment:
Why can shields be enabled/disabled in Kconfig independently from the `SHIELD_AS_LIST` setting?